info.limpet/src/info/limpet/data/store/InMemoryStore.java

```diff
@@ -83,12 +84,20 @@ public boolean remove(Object o)
 			return res;
 		}
 
+		@UIProperty(name="Name", category="Label")
 		@Override
 		public String getName()
 		{
 			return _name;
 		}
 
+		
```

Comment:
@dinkoivanov  - I've tried to produce a property that sits in another category, and that is read only.   It has worked well, IMHO.